pyproject.toml

```diff
@@ -34,22 +34,22 @@ Documentation = "https://ccao-data.github.io/assesspy/"
 
 [project.optional-dependencies]
 dev = [
-  "setuptools >= 61.0",
-  "pytest >= 7.3.0",
-  "pytest-cov >= 4.1.0",
+  "setuptools>=61.0",
+  "pytest>=7.0.0",
+  "pytest-cov>=4.0.0",
```

Comment:
These do need to have a lower bound since they also get resolved using `uv`'s `lowest-direct` option (meaning it will try to install like `pytest==2.0.0`).